addons/web/static/tests/views/form_tests.js

```diff
@@ -9665,6 +9665,399 @@ QUnit.module('Views', {
 
         assert.verifySteps(['mounted', 'willUnmount']);
     });
+
+    QUnit.module('Form View: Quick Edition');
```

Comment:
instead of a sub module, I would prefix all tests with "Quick Edition:". That way, you can still filter in the tests page to only execute those tests. Having submodules in tests files often leads to a mess.